cocli/cmd/corimSubmit.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/apiclient/provisioning"
+)
+
+var (
+	corimFile *string
+	apiServer *string
+	mediaType *string
+)
+
+var submitter ISubmitter = &provisioning.SubmitConfig{}
+
+var corimSubmitCmd = NewCorimSubmitCmd(submitter)
+
+func NewCorimSubmitCmd(submitter ISubmitter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "submit",
+		Short: "submit a CBOR-encoded CoRIM payload",
+		Long: `submit a CBOR-encoded CoRIM payload with supplied api server and media type as inputs
+	
+	cocli corim submit \
+			--corim-file = < CORIM CBOR file > \
+			--api-server = < URI of the endorsement API server >
+			--media-type = < media type associated with CORIM file >
+	`,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if err := checkSubmitArgs(); err != nil {
+				return err
+			}
+
+			// Load the data from the CBOR File
+			data, err := readCorimData(*corimFile)
+			if err != nil {
+				return fmt.Errorf("corim payload read failed: %w", err)
```

Comment:
```suggestion
				return fmt.Errorf("CoRIM payload read failed: %w", err)
```

Reply:
This change will lead to error strings should not be capitalized, lint error